apps/api/app/api/v1/endpoints/workflows.py

```
@@ -479,6 +482,30 @@ async def get_public_workflow(request: Request, workflow_id: str):
         )
 
 
+@router.post(
+    "/workflows/generate-prompt", response_model=GenerateWorkflowPromptResponse
+)
+async def generate_workflow_prompt_endpoint(
+    request: GenerateWorkflowPromptRequest,
+    user: dict = Depends(get_current_user),
```

Ruff (0.15.2)

[warning] Unused function argument: `user`

(ARG001)